vuepress/docs/v6.3.2/docs/curate/ecr-bundle-details.md

```diff
@@ -28,11 +28,11 @@ Here is the basic bundle structure:
 
 ### Bundle Descriptor
 
-The bundle descriptor YAML file aggregates all included components and has the structure shown below. Note that the Page Template feature is pageModels and the Content Template is contentModels. 
+The bundle descriptor YAML file aggregates all included components and has the structure shown below. Note that the Page Template feature is `pageModels` and the Content Template feature is `contentModels`. 
 
 > **Warning**
 >
-> Remember, the file must be named `descriptor.yaml`.
+> The bundle descriptor file must be named `descriptor.yaml`.
 
 **descriptor.yaml.**
```

Comment:
was wondering why this 'descriptor.yaml' was just here.  I understand after looking at the actual page but I think the period confuses things. Also some of the others have 'here is an example of a ..." introducing them which helps.

Reply:
The period is meaningless and can be removed.